ee/vellum_ee/workflows/display/tests/workflow_serialization/test_basic_tool_calling_node_serialization.py

```diff
@@ -130,17 +130,20 @@ def test_serialize_workflow():
                         "type": "JSON",
                         "value": [
                             {
-                                "state": None,
-                                "cache_config": None,
-                                "name": "get_current_weather",
-                                "description": None,
-                                "parameters": {
-                                    "type": "object",
-                                    "properties": {"location": {"type": "string"}, "unit": {"type": "string"}},
-                                    "required": ["location", "unit"],
+                                "function_definition": {
```

Comment:
[nit] would be down to call this just `definition` since the full object is the function. Or `interface`